benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/ExplorationSpaceYamlProtocol.scala

```diff
@@ -29,19 +35,32 @@ object ExplorationSpaceYamlProtocol extends DefaultYamlProtocol {
           yamlObject.getFields(WorkloadKey).headOption.map(_.convertTo[Try[WorkloadExplorationSpace]].get),
           keyString(WorkloadKey))
 
+        services <- deserializationHandler(
+          Option(
+            yamlObject.fields.filterNot(value => value._1.equals(WorkloadKey))
+              .map(value => (value._1.convertTo[String], value._2.convertTo[Try[ServiceExplorationSpace]].get))),
+          keyString(YamlString("(some service)")))
```

Comment:
It is related to reporting errors in parsing, and the issue describing the way to improve this and current limitations is #44